cypress/integration/visitorCanSeeSideArticles.js

```diff
@@ -0,0 +1,25 @@
+describe("Visitor can see side articles", () => {
+  beforeEach(() => {
+    cy.server();
+  });
+  
+  it("successfully", () => {
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:side_articles_shown.json"
+    });
+    cy.visit("/");
+    cy.get("#side-articles-div-2").should("contain", "TestBody2");
+  });
+
+  it("unsuccessfully", () => {
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:side_articles_empty.json"
+    });
+    cy.visit("/");
+    cy.get("#message2").should("contain", "No articles found");
```

Comment:
please come up with something more descriptive for that element `id`

Reply:
changed to error-message. As this div will only contain error messages from the api.